frontend/app/templates/user/following.hbs

```diff
@@ -1,21 +1,20 @@
 <div class="container">
-  {{#each}}
+  {{#each model as |follower|}}
```

Comment:
technically they'd be a `followee` not a `follower`.  Or what would be the opposite of `follower`?  `leader`?